src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 720px;
+  gap: $gap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @include proportions();
+    box-sizing: border-box;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: #ffc0cb;
+      cursor: pointer;
+      transform: translate(0, -20px);
+    }
+  }
+
+  @each $day, $margin in $days {
+    .calendar--start-day--#{$day}
+    &__day:first-child {
+      margin-left: $margin * ($daySize + $gap);
+    };
+  }
+
+  @for $index from 1 to 32 {
+    &__day:nth-of-type(#{$index})::before {
+      content: '#{$index}';
+    }
+  }
+
+  @for $index from 28 through 32 {
```

there couldn't be 32 days
```suggestion
  @for $index from 28 through 31 {
```